docs/sdk/serverless-signing/broadcast-and-collect.md

```diff
@@ -0,0 +1,42 @@
+import FeedbackComponent from "@site/src/pages/feedback.md";
+
+# Broadcast and Collect Within an Action
+
+## Overview
+
+Within a Lit action, you may wish to run a specific operation on all nodes within the network and collect the responses for processing as a set of data from within the node's trusted execution enviorment (TEE). This is helpful if you wish to perform operations over the data as a set. Where all values are grouped and returned to every other node from within the lit action context.
```

Comment:
Copy suggestion:

Current:
"Within a Lit action, you may wish to run a specific operation on all nodes within the network and collect the responses for processing as a set of data from within the node's trusted execution enviorment (TEE). This is helpful if you wish to perform operations over the data as a set. Where all values are grouped and returned to every other node from within the lit action context."

Suggestion:
"'Broadcast and collect' allows you to run an operation on each node in the Lit network, collect their responses, and aggregate those responses into a single data set. This is useful if you'd like the ability to perform additional operations over their responses, such as calculating a median or average. 

When you call this function, the responses from each node will be grouped and then returned back to each node for further processing"

Reply:
>> and aggregate those responses into a single data set.  

Suggestion:  and aggregate those responses into a single data set shared across all of the nodes.